mozilla_django_oidc/utils.py

```diff
@@ -39,3 +39,19 @@ def is_authenticated(user):
     if _less_than_django_1_10:
         return user.is_authenticated()
     return user.is_authenticated
+
+
+def get_oidc_scopes():
```

Comment:
Instead of increasing the complexity by adding an extra function that handles the scopes why don't we just have a config entry with the scopes predefined as a string?